c10/macros/Macros.h

```diff
@@ -217,7 +217,9 @@ using namespace c10::xpu;
 // Unlike C10_ALWAYS_INLINE, C10_ALWAYS_INLINE_ATTRIBUTE can be used
 // on a lambda.
 #if defined(_MSC_VER)
-#define C10_ALWAYS_INLINE_ATTRIBUTE [[msvc::forceinline]]
+// MSVC 14.39 is reasonably recent and doesn't like
```

Comment:
14.39 is a runtime rather than a compiler version...

Reply:
I wonder if it should be https://learn.microsoft.com/en-us/cpp/cpp/attributes?view=msvc-170#msvcforceinline_calls for lambdas